linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_all
+# platform = multi_platform_all,multi_platform_sle
```

Comment:
The `multi_platform_all` includes `multi_platform_sle` which means that there is no need to add this.

Reply:
Ah yes, copy paste error. Fixed.